src/Components/Login/login.js

```diff
@@ -47,31 +48,54 @@ export default function Login() {
         return email.length > 0 && password.length > 0;
     }
 
-    function handleSubmit(event) {
+    async function handleSubmit(event) {
         event.preventDefault();
         const errors = {};
 
-        auth.signInWithEmailAndPassword(email, password).catch((error) => {
-            errors.login = "Email or password is incorrect.";
-            const errorCode = error.code;
+        auth.setPersistence(Auth.Persistence.NONE);
```

Comment:
What does this line do?

Reply:
I forget what the default state was (perhaps local) because I believe it caused issues with needing to sign out like mentioned in this link: https://firebase.google.com/docs/auth/web/auth-state-persistence

Reply:
Changing to SESSION because we want to keep the user logged in on refresh. Good catch